science/freedv-gui/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+categories          science comms
+platforms           darwin macosx
+license             LGPL-2.1
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+description         GUI Application for FreeDV – an open source digital \
+    voice protocol that integrates the modems, codecs, and FEC
+long_description    ${description}
+
+github.setup        drowe67 freedv-gui 9b66784d31102d7d63cc5f67208e24fff95beb66
+version             1.3.1
+checksums           rmd160  222b7b15f4fe8b7c427506a48a20e82fe1c17be4 \
+                    sha256  e8da1d5843cebd9b4c9e53a604eec7c26ce00d2f70c5ebb049bfb893199003af \
+                    size    618399
+revision            0
+
+depends_build-append \
+    port:pkgconfig \
+    port:subversion
```

Comment:
Why is subversion a dependency when the GitHub portgroup is being used?

Reply:
ops; was a reminiscence of using subversion. thank you!